tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -194,7 +222,19 @@ def test_data_invalid(nhsd_apim_proxy_url, property, pointer, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_data_duplicate(nhsd_apim_proxy_url, property, pointer, correlation_id):
     """
-    .. py:function:: Test duplicate message references
+    .. py:function:: Scenario: A user submitting a request with a duplicate attribute \
+        in the request body receives a 400 'Duplicate Value' response
+
```

Comment:
Needs a table with what we consider a duplicate property to be

Reply:
Addressed in request_properties.rst, linked below given/when/then